clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBServerConfig.java

```diff
@@ -372,6 +377,9 @@ public RocksDBServerConfig(VeniceProperties props) {
     this.separateRMDCacheEnabled = props.getBoolean(ROCKSDB_SEPARATE_RMD_CACHE_ENABLED, false);
 
     this.blockBaseFormatVersion = props.getInt(ROCKSDB_BLOCK_BASE_FORMAT_VERSION, 2);
+
+    this.maxLogFileNum = props.getInt(ROCKSDB_MAX_LOG_FILE_NUM, 3);
+    this.maxLogFileSize = props.getSizeInBytes(ROCKSDB_MAX_LOG_FILE_SIZE, 10 * 1024 * 1024); // 10MB;
```

Comment:
What is the granularity of this 3 * 10 MB limit? Is it per Venice partition? Per store-version? Per host? Let's add JavaDoc that mentions this please.

Reply:
Per partition, and updated the javadoc.